### PR TITLE
Rename schema job to be more clear about its purpose

### PR DESCRIPTION
**Pull request by kdelee**
The make target fails when it detects schema changes, not when schema is invalid.

Also update CONTRIBUTING.md to include information about zuul jobs.

##### SUMMARY

There was confusion about what the job was doing that this name change is trying to address.

##### ISSUE TYPE

 - Feature Pull Request
 - Docs Pull Request

##### COMPONENT NAME

- CI

Depends-On: https://github.com/ansible/zuul-jobs/pull/29

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -331,6 +331,23 @@ Sometimes it might take us a while to fully review your PR. We try to keep the `
 
 All submitted PRs will have the linter and unit tests run against them via Zuul, and the status reported in the PR.
 
+## PR Checks ran by Zuul
+Zuul jobs for awx are defined in the [zuul-jobs](https://github.com/ansible/zuul-jobs) repo.
+
+Zuul runs the follwing checks that must pass:
```
Comment (Spredzy) [inline CONTRIBUTING.md]: s/follwing/following

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -331,6 +331,23 @@ Sometimes it might take us a while to fully review your PR. We try to keep the `
...
+1) `tox-awx-api-lint`
```
Comment (Spredzy) [inline CONTRIBUTING.md:338]: Its all about number 1) what about 2) 3) ...

**Review by Spredzy — APPROVED** [review]
Few nits but lgtm

**Comment by Spredzy** [comment]
recheck

**Review by matburt — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (kdelee) [reply CONTRIBUTING.md:338]: I thought that markdown would auto make it pretty

File: CONTRIBUTING.md (hunk 1 above)
Comment (kdelee) [reply CONTRIBUTING.md]: lol

**Review by ryanpetrello — APPROVED** [review]

**Comment by Spredzy** [comment]
recheck

**Pull request merged**